prac_06/programming_language.py

```diff
@@ -0,0 +1,25 @@
+"""Programming Language exercise"""
+
+
+class ProgrammingLanguage:
+    """Represent a ProgrammingLanguage object."""
+
+    def __init__(self, field="", typing="", reflection="", year=0):
```

Comment:
in stead of writing "field", writing"name" is better

Reply:
Noted. Thank you.